src/faucet/views/faucets/mod.rs

```diff
@@ -19,7 +19,7 @@ fn FaucetInput(faucet: RwSignal<FaucetController>) -> impl IntoView {
         <div class="input-container">
             <input
                 type="text"
-                placeholder="Enter target address (Filecoin or Ethereum style)"
+                placeholder="Enter address (Filecoin f... or Ethereum 0x...)"
```

Comment:
It wouldn't be correct for calibnet faucets, the Filecoin addresses for testnets start with `t...`. Additional logic based on the faucet type would be required to deal with this; the required information is already available in the `faucet` parameter, so we either implement that logic or we leave it as it was (or somehow reformulate).